app/code/core/Mage/Tag/Model/Tag.php

```diff
@@ -366,6 +366,7 @@ public function saveRelation($productId, $customerId, $storeId)
             ->setActive(Mage_Tag_Model_Tag_Relation::STATUS_ACTIVE)
             ->setCreatedAt($relationModel->getResource()->formatDate(time()));
 
+        $result = null;
```

Comment:
Return type is string:
```suggestion
        $result = '';
```